src/Button.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import './Button.css';
+import { Link } from 'react-router-dom';
 
 function Button({backToMain}) {
   return (
-   <button onClick={() => backToMain()}> ⬅ Back to Main</button>
+    <Link to="/" key={Date.now()}><button onClick={() => backToMain()}> ⬅ Back to Main</button></Link>
```

Comment:
Awesome! thanks for updating this. 
Do we need the key if there is only one button and we're not doing a map? 
Not that it hurts to have it either way, but I'm curious.

Reply:
Good point, let me check this point out and get back to you!